### PR TITLE
Override alt property for image to use custom alt_text

### PR DESCRIPTION
**Pull request by webbyfox**
- [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: tests/unit/core/test_models.py [hunk 1]
```diff
@@ -1,17 +1,26 @@
...
+    ListPage,
+)
+from domestic.models import DomesticDashboard, DomesticHomePage
 from exportplan.models import ExportPlanDashboardPage
-from domestic.models import DomesticHomePage, DomesticDashboard
 from tests.unit.core import factories
 from .factories import DetailPageFactory
-from wagtail_factories import ImageFactory
-
-from django.core.exceptions import ValidationError
```
Comment (stevejalim) [inline tests/unit/core/test_models.py:14]: Thanks for cleaning up the import ordering 💅

File: tests/unit/core/test_models.py [hunk 2]
```diff
@@ -111,3 +120,24 @@ def test_validation_kick_for_multiple_hero_image(self):
...
+
+
+class TestImageAltRendition(TestCase, WagtailTestUtils):
+    def setUp(self):
+        self.login()
+        root_collection = Collection.objects.create(name='Root', depth=0)
+        great_image_collection = root_collection.add_child(name="Great Images")
+
+        # Create an image with alt text
+        AltTextImage = get_image_model()
```
Comment (stevejalim) [inline tests/unit/core/test_models.py:132]: (Not a change request 😄 )

It'd be good to improve this naming of `AltTextImage` before things go too far - `GreatImage` or something more generic will be less misleading if the `AltTextImage` also ends up having, for example, a custom `caption` field too

**Review by stevejalim — APPROVED** [review]
LGTM! Thanks for the solution.

One nit and one comment - but neither is a blocker

File: tests/unit/core/test_models.py (hunk 2 above)
Comment (webbyfox) [reply tests/unit/core/test_models.py]: Yeah, Agreed. although any model name changes I would like to do as separate PR/ticket

**Review by bobmeredith — APPROVED** [review]
LGTM

**Pull request merged**